packages/cli/src/commands/setup/cache/templates/memcached.ts.template

```diff
@@ -0,0 +1,21 @@
+import { createCache, MemcachedClient } from '@redwoodjs/api/cache'
+
+import { logger } from './logger'
+
+const memJsFormattedLogger = {
+  log: (msg: string) => logger.error(msg),
+}
+
+let client: MemcachedClient
+try {
+  client = new MemcachedClient('localhost:11211', {
```

Comment:
Should the host be in an environment variable?

Reply:
I thought about that, but I didn't want to go down the road of the generator writing something into your own`.env` file (pretty sure we don't do that anywhere else?). And it shouldn't go into `.env.defaults` because that's committed and someone could end up putting credentials for a remote system if they're not running the service on `localhost` and not even be aware of it. I could do a post-install note, but since no one reads those they'll be confused as to why they're getting a bunch of errors as soon as they run `yarn rw dev`. So I fell back to just hardcoding `localhost` right in there. 😐

Reply:
I think we do that with dbAuth actually! When you setup dbAuth, it adds the session secret automatically.